### ISSUE TITLE
if use_fork is false, this should be ignored

### ISSUE THREAD

**Issue by hsliuustc** [alasdairtran/fourierflow#3]
https://github.com/alasdairtran/fourierflow/blob/97e6cfb0848e44d3a7bc1d063b1ab86bc4c603ee/fourierflow/modules/fourier_2d_factorized_parallel.py#L218

**Comment by alasdairtran** [comment]
Yeah you're right! I had a go at running the experiment with `avg_outs` set to False, and it makes no difference to the loss (which makes sense since it's just a linear scaling factor, which the projection layer can easily learn). I'll probably remove this scaling altogether in the next iteration of experiments.
